packages/@react-stately/combobox/test/useComboBoxState.test.js

```diff
@@ -204,21 +203,32 @@ describe('useComboBoxState tests', function () {
       expect(result.current.selectionManager.selectionMode).toBe('single');
       expect(result.current.selectionManager.selectedKeys.size).toBe(0);
 
-      act(() => {result.current.open();});
       act(() => result.current.selectionManager.replaceSelection('1'));
       expect(result.current.selectionManager.selectedKeys).toContain('1');
       expect(result.current.selectionManager.selectedKeys).not.toContain('0');
       expect(onSelectionChange).toHaveBeenCalledWith('1');
     });
 
-    it('won\'t perform replace a selection if the combobox is closed', function () {
-      // This case covers if a option in the menu is clicked while the menu is closing
-      let initialProps = {...defaultProps};
+    it('won\'t update the returned collection if the combobox is closed', function () {
```

Comment:
should have a controlled test as well?

Reply:
sure I can add a controlled `items` test